infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -70,6 +69,14 @@ public void addServer(AbstractServer server) {
         throw new UnsupportedOperationException("No longer supported");
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<AbstractServer> getServers() {
+        return ImmutableList.copyOf(servers);
```

Comment:
What's the point of returning immutable copies, why not just make the class field immutable and return that?